tests/b2b-mp/sapi/tests/checkout/B2B-MP-SAPI6-checkout.js

```diff
@@ -0,0 +1,24 @@
+import { SharedCheckoutScenario } from "../../../../cross-product/sapi/scenarios/checkout/shared-checkout-scenario.js";
+import { loadDefaultOptions } from "../../../../../lib/utils.js";
+
+export const options = loadDefaultOptions();
+options.scenarios = {
+    SAPI6_Checkout: {
+        exec: 'executeCheckoutScenario',
+        executor: 'shared-iterations',
+        env: {
+            sku: __ENV.sku || '100429'
+        },
+        tags: {
+            testId: 'SAPI6',
+            testGroup: 'Checkout',
+        },
+        iterations: 1
+    },
+};
+
+const checkoutScenario = new SharedCheckoutScenario('B2B_MP');
+
+export function executeCheckoutScenario() {
+    checkoutScenario.execute();
+}
```

Comment:
We are not testing checkout, the tests names are `Add 1 item to cart` and `Add 70 items to cart`. The tests, scenarios, options, methods must be renamed and reworked accordingly